Makefile

```diff
@@ -46,13 +46,14 @@ build:: tfgen provider
 		rm ./bin/setup.py.bak && \
 		cd ./bin && $(PYTHON) setup.py build sdist
 	cd ${PACKDIR}/dotnet/ && \
-    echo "${VERSION:v%=%}" >version.txt && \
+		echo "${VERSION:v%=%}" >version.txt && \
 		dotnet build /p:Version=${DOTNET_VERSION}
 
 tfgen::
 	go install -ldflags "-X github.com/pulumi/pulumi-${PACK}/pkg/version.Version=${VERSION}" ${PROJECT}/cmd/${TFGEN}
 
 provider::
+	packr2
```

Comment:
Do we not need a file that forces the generated `packrd` module to be loaded?  I had this here: https://github.com/pulumi/pulumi-aiven/blob/lukehoban/packr/aiven-packr.go.  I don't fully understand how it works - but would have expected we need something in our root `aiven` module which imports this.  (or in the `cmd/${PROVIDER}` next to `main.go`? Actually that is probably a better place for it if its only needed at runtime not at tfgen time.).